### ISSUE TITLE
Blank screen when I open zimpedia

### ISSUE THREAD

**Issue by carmenfdezb** [mkiol/Zimpedia#31]
I get next error when I open zimpedia from Terminal:
```
,---
| Sailfish OS 3.1.0.11 (Seitseminen)
'---
[nemo@Sailfish ~]$ harbour-zimpedia
[D] unknown:0 - Using Wayland-EGL
[W] unknown:0 - "Couldn't load translation for locale C from /usr/share/harbour-zimpedia/translations"
[D] unknown:0 - Scanning file: "/media/sdcard/TarjetaSD/Diccionario/wikipedia_es_all_novid_2018-04.zim"
[D] unknown:0 - ZIM major version: 5
[D] unknown:0 - ZIM minor version: 0
[D] unknown:0 - multipart: false
[W] modifyPageStack:113 - Error while modifying page stack: Error: Error while loading page: file:///usr/share/harbour-zimpedia/qml/FilesPage.qml:81 Cannot assign to non-existent property "text"

[W] unknown:483 - file:///usr/lib/qt5/qml/Sailfish/Silica/PageStack.js:483: Error: Error while loading page: file:///usr/share/harbour-zimpedia/qml/FilesPage.qml:81 Cannot assign to non-existent property "text"

[W] unknown:0 - Cannot open bookmarks file
[W] unknown:0 - No items
```
And zimpedia shows blank screen

**Comment by mkiol** [comment]
Thank you for reporting. Indeed there is a naming conflict.

The fix: 8d69dc8598adf5f0c8fae36bdff12be78abfb3dc

Fix is included in the 2.2.5 release (available in OpenRepos)

**Comment by carmenfdezb** [comment]
It works perfect now. Thank you!!
